disk.go

```diff
@@ -0,0 +1,26 @@
+package skewer
+
+// HasSCSISupport determines if a SKU supports SCSI disk controller type.
+// If no disk controller types are declared, it assumes SCSI is supported for backward compatibility.
+func (s *SKU) HasSCSISupport() bool {
+	declaresSCSI := s.HasCapabilityWithSeparator(DiskControllerTypes, DiskControllerSCSI)
+	declaresNVMe := s.HasCapabilityWithSeparator(DiskControllerTypes, DiskControllerNVMe)
```

Comment:
declaresNVMe is no longer needs, can be replaced with HasNVMESupport()

Reply:
ack implemented this way.